pyproject.toml

```diff
@@ -35,15 +35,17 @@ emoji = "^1.7.0"
 tensorly = "^0.7.0"
 rasterio = "^1.3.0"
 ipywidgets = "^8.0"
-decoupler = "^1.2.0"
+decoupler = "1.4.0"
 MiniSom = "^2.3.0"
 kneed = "^0.8.1"
 adjustText = "^0.7.3"
 sparse = "^0.13.0"
 pandas = "^1.5.3"
-xgboost = "1.4.0"
+xgboost = "2.0.0"
```

Comment:
xgboost 2.0.0 seems to work for `bentot.tl.lp` (for loading random forest model)